jax/_src/plugin_utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The JAX Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pathlib
+import glob
+
+# Code to suggest plugins that should be installed.
```

Comment:
nit: this comment should still be in xla_bridge.py where it was before (i.e. above `_suggest_missing_backends`), not in this file. The suggesting code is still in xla_bridge.